javascript/packages/orchestrator/src/chainSpec.ts

```diff
@@ -432,6 +432,11 @@ export async function addParachainToGenesis(
     else if (runtimeConfig.parachainsParas) {
       paras = runtimeConfig.parachainsParas.paras;
     }
+    // The config may not contain paras. Since chainspec allows to contain the RuntimeGenesisConfig patch we can inject it.
+    else {
+      runtimeConfig.paras = { paras: [] };
+      paras = [];
```

Comment:
@wirednkod hm, is this correct? If I read code correctly in the line 433 and 429 `paras` is the ref to existing `runtimeConfig.paras.paras` field. Why here you suggested sth different?

Reply:
This won't work, I'll revert it.